routes/databaseRoutes.js

```diff
@@ -0,0 +1,171 @@
+require("dotenv").config({ path: "../.env" });
+
+const dbParams = require("../lib/db");
+const { Pool } = require("pg");
+const format = require("pg-format");
+
+const db = new Pool(dbParams);
+
+const getUserByEmail = (email) => {
+  const queryString = `SELECT * FROM users WHERE email = $1;`;
+
+  return db
+    .query(queryString, [email])
+    .then((res) => {
+      if (res.rows.length < 1) {
+        return null;
+      }
+      return res.rows[0];
+    })
+    .catch((err) => console.log(err));
+};
+
+exports.getUserByEmail = getUserByEmail;
+
+const getUserById = (id) => {
+  const queryString = `SELECT * FROM users WHERE id = $1;`;
+
+  return db
+    .query(queryString, [id])
+    .then((res) => {
+      if (res.rows.length < 1) {
+        return null;
+      }
+      return res.rows[0];
+    })
+    .catch((err) => console.log(err));
+};
+exports.getUserById = getUserById;
+
+const addUser = (user) => {
+  const queryString = `
+  INSERT INTO users (
+    first_name, last_name, phone_number, email, password
+  ) VALUES (
+    $1, $2, $3, $4, $5
+  ) RETURNING *;
+  `;
+  const values = [
+    user.firstName,
+    user.lastName,
+    user.phoneNumber,
+    user.email,
+    user.password,
+  ];
+
+  return db
+    .query(queryString, values)
+    .then((res) => {
+      return res.rows[0];
+    })
+    .catch((err) => console.log(err));
+};
+
+exports.addUser = addUser;
+
+const getAllOrders = (user_id, limit = 10) => {
```

Comment:
We may need getUserOrders for user, and getOpenOrders & getHistoryOrders for admin?